bin/gql-gen.ts

```diff
@@ -40,53 +40,54 @@ async function main(): Promise<void> {
         const cwd = process.cwd();
         const lib = libDir(cwd);
 
+        // patch up codegen template to create optional properties
+        const codegenTemplate = path.join(cwd, "node_modules", "graphql-codegen-typescript-client", "dist", "index.js");
+        if (await fs.pathExists(codegenTemplate)) {
+            let contents = (await fs.readFile(codegenTemplate)).toString();
+            contents = contents.replace(
+                /{{ name }}: {{ convertedFieldType/i,
+                "{{ name }}?: {{ convertedFieldType");
```

Comment:
If this regular expression is case insensitive, shouldn't the replacement maintain the same case?

```typescript
                /{{ (name) }}: {{ (convertedfieldtype)/i,
                "{{ $1 }}?: {{ $2");
```

Reply:
I’ll remove the i in the regex.